pkg/k8s/node_utils.go

```diff
@@ -6,12 +6,34 @@ import (
 	"strings"
 )
 
+const (
+	toBeDeletedByCATaint = "ToBeDeletedByClusterAutoscaler"
+)
+
 // IsNodeReady returns whether node is ready.
 func IsNodeReady(node *corev1.Node) bool {
 	nodeReadyCond := GetNodeCondition(node, corev1.NodeReady)
 	return nodeReadyCond != nil && nodeReadyCond.Status == corev1.ConditionTrue
 }
 
+// IsNodeSuitableAsTrafficProxy check whether node is suitable as a traffic proxy.
+// mimic the logic of serviceController: https://github.com/kubernetes/kubernetes/blob/b6b494b4484b51df8dc6b692fab234573da30ab4/pkg/controller/service/controller.go#L605
+func IsNodeSuitableAsTrafficProxy(node *corev1.Node) bool {
+	if node.Spec.Unschedulable {
```

Comment:
we shouldn't check for Unschedulable anymore.(it's removed from latest k8s of whether it should be a critera for LB node pool)